content/02.main-text.md

```diff
@@ -68,7 +91,7 @@ After four rounds of reviews and pull request edits, a maintainer merged the cha
 We found that this workflow was an effective compromise between fully unrestricted editing and a more heavily-structured approach that limited the authors or the sections they could edit.
 In addition, authors are associated with their commits, which makes it easy for contributors to receive credit for their work and helps prevent ghostwriting [@doi:10.1371/journal.pmed.1000023].
 Figure @fig:contrib and the GitHub [contributors page](https://github.com/greenelab/deep-review/graphs/contributors) summarize all edits and commits from each author, providing aggregated information that is not available on other collaborative writing platforms.
-Because our writing process, like others backed by the open git version control system (including Overleaf and Authorea), tracks the complete commit history, it also enables detailed retrospective contribution analysis.
+Because our writing process tracks the complete commit history through git commits, it enables detailed retrospective contribution analysis.
```

Comment:
Do we want to say "commit" twice in this sentence?

Reply:
Fixed in c0e2a22